docs/advanced_examples/ExperimentPrivacyTreePaper.ipynb

```diff
@@ -105,89 +206,114 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Average precision: 0.8590584502269576\n",
-      "Average precision (fp32): 0.856742055225908\n",
-      "Average precision: 0.8859057196962661\n",
-      "Average precision (fp32): 0.8978489602543138\n",
-      "Average precision: 0.8533618889112272\n",
-      "Average precision (fp32): 0.8702812578992057\n",
-      "Average precision: 0.5257920855944928\n",
-      "Average precision (fp32): 0.5313028959377616\n",
-      "Average precision: 0.5311883185588149\n",
-      "Average precision (fp32): 0.5435889125308891\n",
-      "Average precision: 0.4736968702820955\n",
-      "Average precision (fp32): 0.4878108468555631\n",
-      "Average precision: 1.0\n",
-      "Average precision (fp32): 1.0\n",
-      "Average precision: 1.0\n",
-      "Average precision (fp32): 1.0\n",
-      "Average precision: 1.0\n",
-      "Average precision (fp32): 1.0\n"
+      "DecisionTreeClassifier on spambase (#features: 57) -> Acc: 0.9103, Acc (fp32): 0.9035, FHE inference time: 1.59s\n",
+      "XGBClassifier on spambase (#features: 57) -> Acc: 0.9313, Acc (fp32): 0.9362, FHE inference time: 2.01s\n",
+      "RandomForestClassifier on spambase (#features: 57) -> Acc: 0.9089, Acc (fp32): 0.9184, FHE inference time: 6.88s\n",
```

Comment:
I agree, this print is not visually appealing.

I can suggest to reformat the print, like:


```
        shift = 52 - len(f"{cls.__name__} on {key_dataset}")

        print(
            f"{cls.__name__} on {key_dataset} {'->':>{shift}} Acc: {np.mean(scores['accuracy']):.4f}, "
            f"Acc (fp32): {np.mean(scores_fp32['accuracy']):.4f}, "
            f"FHE inference time: {scores['inference_time']:.2f}s"
        )

```
![image](https://github.com/zama-ai/concrete-ml/assets/14870263/95f8c164-777d-4e52-8274-0f8da6607831)